README.md

```diff
@@ -99,3 +99,46 @@ $ nvidia-docker run \
     --log-opt max-file=3 \
     codalab/competitions-v2-compute-worker:nvidia 
 ```
+
+# Worker management
+
+Outside of docker containers install [Fabric](http://fabfile.org/) like so:
+
+```bash
+pip install fab-classic
```

Comment:
Any way we can do this with a docker container instead of directly on our machines? at the very least, should probably specify a version number.

Reply:
good call, adding